--- openid.php	2009-03-20 03:15:45.000000000 +0100
+++ openid-patched.php	2009-04-19 18:45:23.000000000 +0200
@@ -111,7 +111,8 @@
  */
 function get_user_by_openid($url) {
 	global $wpdb;
-	return $wpdb->get_var( wpdb_prepare('SELECT user_id FROM '.openid_identity_table().' WHERE url = %s', $url) );
+	$id = $wpdb->get_var( wpdb_prepare('SELECT user_id FROM '.openid_identity_table().' WHERE url = %s', $url) );
+	return apply_filters('openid_get_user_by_openid', $url, $id);
 }
 
 
