--- common.php	2009-03-20 03:09:15.000000000 +0100
+++ common.php.patched	2009-07-07 10:53:10.000000000 +0200
@@ -40,7 +40,6 @@
 }
 
 
-
 /**
  * Set the textdomain for this plugin so we can support localizations.
  */
@@ -92,9 +91,7 @@
 	static $consumer;
 
 	if (!$consumer) {
-		set_include_path( dirname(__FILE__) . PATH_SEPARATOR . get_include_path() );
-		require_once 'Auth/OpenID/Consumer.php';
-		restore_include_path();
+		openid_require_library('Auth/OpenID/Consumer.php');
 
 		$store = openid_getStore();
 		$consumer = new Auth_OpenID_Consumer($store);
@@ -404,8 +401,7 @@
 
 		if (is_email($url)) {
 			$_SESSION['openid_login_email'] = $url;
-			set_include_path( dirname(__FILE__) . PATH_SEPARATOR . get_include_path() );
-			require_once 'Auth/Yadis/Email.php';
+			openid_require_library('Auth/Yadis/Email.php');
 			$mapped_url = Auth_Yadis_Email_getID($url, trailingslashit(get_option('home')));
 			if ($mapped_url) {
 				$url = $mapped_url;
@@ -493,9 +489,7 @@
  */
 function openid_add_ax_extension($extensions, $auth_request) {
 	if(!get_user_by_openid($auth_request->endpoint->claimed_id)) {
-		set_include_path( dirname(__FILE__) . PATH_SEPARATOR . get_include_path() );
-		require_once('Auth/OpenID/AX.php');
-		restore_include_path();
+		openid_require_library('Auth/OpenID/AX.php');
 
 		if ($auth_request->endpoint->usesExtension(Auth_OpenID_AX_NS_URI)) {
 			$ax_request = new Auth_OpenID_AX_FetchRequest();
@@ -516,9 +510,7 @@
  */
 function openid_add_sreg_extension($extensions, $auth_request) {
 	if(!get_user_by_openid($auth_request->endpoint->claimed_id)) {
-		set_include_path( dirname(__FILE__) . PATH_SEPARATOR . get_include_path() );
-		require_once('Auth/OpenID/SReg.php');
-		restore_include_path();
+		openid_require_library('Auth/OpenID/SReg.php');
 
 		if ($auth_request->endpoint->usesExtension(Auth_OpenID_SREG_NS_URI_1_0) || $auth_request->endpoint->usesExtension(Auth_OpenID_SREG_NS_URI_1_1)) {
 			$extensions[] = Auth_OpenID_SRegRequest::build(array(),array('nickname','email','fullname'));
@@ -698,9 +690,7 @@
  * @see get_user_data
  */
 function openid_get_user_data_ax($data, $identity_url) {
-	set_include_path( dirname(__FILE__) . PATH_SEPARATOR . get_include_path() );
-	require_once('Auth/OpenID/AX.php');
-	restore_include_path();
+	openid_require_library('Auth/OpenID/AX.php');
 
 	$response = openid_response();
 	$ax = Auth_OpenID_AX_FetchResponse::fromSuccessResponse($response);
@@ -739,7 +729,7 @@
  * @see get_user_data
  */
 function openid_get_user_data_sreg($data, $identity_url) {
-	require_once(dirname(__FILE__) . '/Auth/OpenID/SReg.php');
+	openid_require_library('Auth/OpenID/SReg.php');
 	$response = openid_response();
 	$sreg_resp = Auth_OpenID_SRegResponse::fromSuccessResponse($response);
 	$sreg = $sreg_resp->contents();
