--- inc/parserutils.php	2010-10-07 11:01:02.000000000 +0200
+++ inc/parserutils.php.new	2010-10-07 11:01:08.000000000 +0200
@@ -634,14 +634,15 @@
     $cache = getCacheName($language.$code,".code");
     $ctime = @filemtime($cache);
     if($ctime && !$_REQUEST['purge'] &&
-            $ctime > filemtime(DOKU_INC.'inc/geshi.php') &&                 // geshi changed
-            $ctime > @filemtime(DOKU_INC.'inc/geshi/'.$language.'.php') &&  // language syntax definition changed
+            $ctime > filemtime(LD_LIB_DIR.'/geshi/geshi.php') &&                 // geshi changed
+            $ctime > @filemtime(LD_LIB_DIR.'/geshi/geshi/'.$language.'.php') &&  // language syntax definition changed
             $ctime > filemtime(reset($config_cascade['main']['default']))){ // dokuwiki changed
         $highlighted_code = io_readFile($cache, false);
 
     } else {
 
-        $geshi = new GeSHi($code, $language, DOKU_INC . 'inc/geshi');
+        require_once(LD_LIB_DIR . '/geshi/geshi.php');
+        $geshi = new GeSHi($code, $language, LD_LIB_DIR . '/geshi/geshi');
         $geshi->set_encoding('utf-8');
         $geshi->enable_classes();
         $geshi->set_header_type(GESHI_HEADER_PRE);
