--- inc/init.php	2011-03-17 22:44:13.000000000 +0100
+++ inc/init.php.new	2011-03-17 22:44:47.000000000 +0100
@@ -20,6 +20,8 @@
 // define the include path
 if(!defined('DOKU_INC')) define('DOKU_INC',fullpath(dirname(__FILE__).'/../').'/');
 
+require_once(DOKU_INC.'dist/config.php');
+
 // define Plugin dir
 if(!defined('DOKU_PLUGIN'))  define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
 
@@ -95,6 +97,8 @@
     }
 }
 
+require_once(DOKU_INC.'dist/prepend.php');
+
 // set timezone (as in pre 5.3.0 days)
 date_default_timezone_set(@date_default_timezone_get());
 
