--- registration/1.0/index.php.original	2010-02-02 12:29:00.000000000 +0100
+++ registration/1.0/index.php	2010-02-02 12:32:02.000000000 +0100
@@ -37,6 +37,8 @@
 #
 # ***** END LICENSE BLOCK *****
 
+	require_once(dirname(__FILE__) . '/../../dist/prepend.php');
+
 	require_once 'weave_user_constants.php';
 	require_once 'utils.php';
 	require_once 'weave_user/' . WEAVE_AUTH_ENGINE . '.php';
@@ -46,6 +48,10 @@
 	else if (!empty($_SERVER['ORIG_PATH_INFO'])) 
 		$path = $_SERVER['ORIG_PATH_INFO'];
 
+	if (function_exists('ld_weave_registration_path')) {
+		$path = ld_weave_registration_path();	
+	}
+	
 	$path = substr($path, 1); #chop the lead slash
 	list($url_user, $action) = explode('/', strtolower($path) . '/');
 	
