--- sync/1.0/weave_user/mysql.php.orig	2010-03-01 14:34:21.000000000 +0100
+++ sync/1.0/weave_user/mysql.php	2010-03-01 14:34:21.000000000 +0100
@@ -102,6 +102,9 @@
 		try
 		{
 			$select_stmt = 'select id, status, alert from users where username = ? and md5 = ?';
+			if (function_exists('ld_weave_prepare_sql')) {
+				$select_stmt = ld_weave_prepare_sql($select_stmt);
+			}
 			$sth = $this->_dbh->prepare($select_stmt);
 			$phash = $this->hash_password($password);
