--- zip/class.unzip.php.orig	2010-06-29 14:23:33.000000000 +0200
+++ zip/class.unzip.php	2010-06-29 14:23:33.000000000 +0200
@@ -101,14 +101,14 @@
 			throw new Exception(sprintf(__('Trying to unzip a folder name %s'),$file_name));
 		}
 		
-		if (!$details['uncompressed_size']) {
-			return $this->putContent('',$target);
-		}
-		
 		if ($target) {
 			$this->testTargetDir(dirname($target));
 		}
 		
+		if (!$details['uncompressed_size']) {
+			return $this->putContent('',$target);
+		}
+		
 		fseek($this->fp(),$details['contents_start_offset']);
 		
 		$this->memoryAllocate($details['compressed_size']);
@@ -205,7 +205,7 @@
 	protected function putContent($content,$target=false)
 	{
 		if ($target) {
-			$r = @file_put_contents($target,$content);
+			$r = file_put_contents($target,$content);
 			if ($r === false) {
 				throw new Exception(__('Unable to write destination file.'));
 			}
