--- codecolorer-core.php	2009-10-21 17:36:28.000000000 +0200
+++ codecolorer-core.php.modified	2009-10-21 17:36:55.000000000 +0200
@@ -282,8 +282,7 @@
 
       # Maybe GeSHi has been loaded by some another plugin?
       if (!class_exists('GeSHi')) {
-        if (!file_exists("$path/lib/geshi.php")) return null;
-        require_once("$path/lib/geshi.php");
+        require_once("geshi/geshi.php");
       } else {
         $instance->geshiExternal = true;
       }
