--- wp-admin/menu-header.php	2010-10-10 21:26:04.000000000 +0200
+++ wp-admin/menu-header.php.new	2010-10-10 21:26:13.000000000 +0200
@@ -47,6 +47,8 @@
 		if ( !empty($submenu[$item[2]]) )
 			$class[] = 'wp-has-submenu';
 
+		do_action('wp_menu_output');
+
 		if ( ( $parent_file && $item[2] == $parent_file ) || ( false === strpos($parent_file, '?') && $self == $item[2] ) ) {
 			if ( !empty($submenu[$item[2]]) )
 				$class[] = 'wp-has-current-submenu wp-menu-open';
